tests/api/v2/datadog/scenarios_test.go

```diff
@@ -303,6 +315,35 @@ func team(t gobdd.StepTest, ctx gobdd.Context) {
 
 	tests.GetData(ctx)["team"] = response
 }
+func deleteIncident(ctx context.Context, incidentID string) {
+	_, err := Client(ctx).IncidentsApi.DeleteIncident(ctx, incidentID).Execute()
+	if err == nil {
+		return
+	}
+	log.Printf("Error deleting incident: %v, Another test may have already deleted this incident: %s", incidentID, err.Error())
+}
+
+func incident(t gobdd.StepTest, ctx gobdd.Context) {
+	client := Client(tests.GetCtx(ctx))
+	client.GetConfig().SetUnstableOperationEnabled("CreateIncident", true)
```

Comment:
Can you add `And operation "CreateIncident" enabled` step in feature definition?

Reply:
This should be fixed now